### DISCUSSION TITLE
Delete event from event page

### DISCUSSION THREAD

**Discussion by charlesrr** [liedekef/events-made-easy#113] [Ideas]
Hi. Thanks for soldiering through and moving to github! All installed and working fine.

I'm using front end submit so that event creators don't have to mess about in the back end of wordpress. However, sometimes they need to delete an event, which I believe can only be done in wordpress admin and someone recently deleted a bunch of other people's events by accident #facepalm. It would be cool if there was a way to display a button on an event page which deletes the event and make that button only visible to the event creator (and sends emails about cancellation to the attendees).

**Comment by liedekef** [comment]
Deleting an event is possible by those with correct rights (events admins, event creator, event contact person). If one of those accidentally deletes an event, then it is their responsibility ...
But events are never deleted immediately, but put in trash bin, so recovery is possible.

**Reply by charlesrr** [reply]
Yeah, numpties! Just as an idea for the future, if they were able to delete their own events from a front end button, it would be cool - they aren't techy people. (sorry, by "in wordpress admin", I meant the backend of wordpress, not the user level).

**Reply by liedekef** [reply]
Deleting events is protected by checking permissions, being in the backend and a nounce ... putting that in the frontend would decrease security in a way that is not acceptable, so that is not going to happen. You can always point them to the event edit-link (where the delete button is then visible as well), that link is not protected on its own.
